### PR TITLE
Bugfix/refresh group

### PR DESCRIPTION
**Pull request by francoisph**

### CODE REVIEW

File: client/src/model/mixins/UserDataWatcher.js [hunk 1]
```diff
@@ -110,7 +110,10 @@ define( [
          * Checks the userData for a change, get the watched value and call the userDataChanged method to notify
          * a mixin user of the change.
          */
-        _checkUserData: function () {
+        _checkUserData: function ( path ) {
+            if ( path && this.userDataPath.indexOf( path ) < 0 ) {
```
Comment (ubald) [inline client/src/model/mixins/UserDataWatcher.js]: I would keep `== 0` because you dont want to match paths that don't start with what you defined

Comment (ubald) [inline client/src/model/mixins/UserDataWatcher.js]: Also, `this.userDataPath` and `path` should be inverted for the `indexOf` to work

**Comment by ubald** [comment]
Quiddity.js line 73 should read `this.trigger( 'change:userData', path );`

**Comment by francoisph** [comment]
@ubald update code review

**Comment by ubald** [comment]
Moved to gitlab

**Pull request closed without merging**